src/components/ProductsList/ListElement/ListElementStyle.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components'
+
+// language=SCSS prefix=dummy{ suffix=}
+const ListElement = styled.li`
+    font-size: 20px;
+    margin: 20px 0;
+    padding: 20px;
+    border: black 2px solid;
+    height: 100px;
+    
+    &:hover {
```

Comment:
You mean height? I know it's bad approach also because of responsiveness but all styles are to rewrite. I was doing it fastest way because at the beginning I haven't plan how it could look ;)